msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,7 +1,14 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/1.5.4">
-
-  <PropertyGroup>
+  <Target Name="MacOSBuild" BeforeTargets="Build">
+    <Message Text="MAC_OS_BUILD = $(MAC_OS_BUILD)" Importance="High" />
+  </Target>
+  <PropertyGroup Condition="'$(MAC_OS_BUILD)' == ''">
     <TargetFrameworks>net45;netstandard1.3;uap10.0;MonoAndroid8.1;Xamarin.iOS10;netcoreapp1.0</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(MAC_OS_BUILD)' != ''">
+    <TargetFrameworks>netstandard1.3;Xamarin.iOS10</TargetFrameworks>
```

Comment:
>netstandard1.3;Xamarin.iOS10 [](start = 22, length = 28)

see general comment on which targets to include for mac

Reply:
see comment elsewhere

---
In reply to: [235163880](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1418#discussion_r235163880) [](ancestors = 235163880)